### PR TITLE
New layout Guru-PR

### PR DESCRIPTION
**Pull request by nandomoreirame**
Hi guys!
Look new redesign of the site.. \o/

thanks..

guru-pr-home

guru-pr-events

guru-pr-event-show

### CODE REVIEW

**Comment by vyper** [comment]
Cool :metal:

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.2.3'
+# ruby '2.2.3'
```
Comment (adbatista) [inline Gemfile]: This change is not related with the PR, could you undo?

Comment (nandomoreirame) [inline Gemfile]: OK, corrected..
sorry :)

**Comment by vyper** [comment]
Hi @nandomoreirame! Congrats for your job!

But, I found some problems:
- the admin menu is broken when I logged with admin user
- the any admin pages (events and users) are with layout broken

For test this:
1. log with user
2. open `rails console`
3. give admin access to last user `User.last.update_attribute, :admin, true`
4. reopen site

# Broken admin menu

## Before
screen shot 2016-01-06 at 9 00 31 am

## Now
screen shot 2016-01-06 at 9 02 27 am


# Admin pages

## Before
screen shot 2016-01-06 at 9 03 40 am
screen shot 2016-01-06 at 9 04 04 am
screen shot 2016-01-06 at 9 04 21 am


## Now
screen shot 2016-01-06 at 9 04 40 am
screen shot 2016-01-06 at 9 05 08 am
screen shot 2016-01-06 at 9 05 26 am

**Comment by nandomoreirame** [comment]
Hi guys!
Please, sorry for my delay :)

All corrections have been completed. Please, check.

**Comment by vyper** [comment]
Awesome @nandomoreirame! I opened a issue for improve responsive layout #7! 

I accept this PR! <3 :clap:

Congrats! Thanks!
